form.js

```diff
@@ -0,0 +1,15 @@
+const emailAddress = document.getElementById('mail');
+const validatorMessage = document.getElementById('validator-message');
+const btn1 = document.querySelector('.get2');
+const form = document.querySelector('#form');
+
+btn1.addEventListener('click', (e) => {
+  if (emailAddress.value !== emailAddress.value.toLowerCase()) {
+    validatorMessage.textContent = 'pleas the input have to be in lowercase';
+    e.preventDefault();
+    console.log('helloooo');
```

Comment:
[OPTIONAL] Just a tip, if you remove unnecessary console logs your code will look cleaner.

Reply:
@smunozmo thank you very much really appreciate